haas/cli.py

```diff
@@ -235,9 +169,11 @@ def check_status_code(response):
     else:
         sys.stdout.write(response.text + "\n")
 
-
+# Function object_url should be DELETED.
 # TODO: This function's name is no longer very accurate.  As soon as it is
 # safe, we should change it to something more generic.
+
+
 def object_url(*args):
     # Prefer an environmental variable for getting the endpoint if available.
     url = os.environ.get('HAAS_ENDPOINT')
```

Comment:
that and subsequent four functions were there prior to this PR. I do not see the point of changing their location if that is not going to add any value to the code. I have specified that they will soon be deleted before `def object_url` and after `def do_delete`